holidays/countries/fiji.py

```diff
@@ -0,0 +1,128 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.calendars.gregorian import SAT, SUN, OCT, NOV
+from holidays.constants import PUBLIC, WORKDAY
+from holidays.groups import ChristianHolidays, InternationalHolidays, IslamicHolidays
+from holidays.observed_holiday_base import (
+    ObservedHolidayBase,
+    SUN_TO_NEXT_MON,
+    SAT_SUN_TO_NEXT_MON_TUE,
+    TUE_TO_PREV_MON,
+)
```

Comment:
`SUN_TO_NEXT_MON` is already included in `SAT_SUN_TO_NEXT_MON_TUE` so we can simplify.
```suggestion
from holidays.observed_holiday_base import (
    ObservedHolidayBase,
    SAT_SUN_TO_NEXT_MON_TUE,
    TUE_TO_PREV_MON,
)
```